### ISSUE TITLE
Failed to start up socket

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#91]
```
Operating system and version (32/64bit) :Win 8 64-bit
.Net Framework version :4.5
Office name and version(32/64bit) :2013 32-bit
Browser name and version :Firefox 32.0, 31.0
SeleniumWrapper version :1.0.19.0


Public Sub a()

    Dim Selenium As New SeleniumWrapper.WebDriver
    Dim keys As New SeleniumWrapper.keys
    Dim BaseUrl As String

    BaseUrl = "http://www.google.com"

'   Web Browser
    Selenium.setProfile "Selenium"
    Selenium.Start "firefox", BaseUrl 'Error line
    Selenium.Open "/"
    Selenium.Wait (1000)
'   selenium.setImplicitWait 5000

'    ...
    Selenium.Stop
End Sub

err.Number: -2146233088 
err.Description: Failed to start up socket within 45000 ms. Attempted to 
connect to the following addresses: 127.0.0.1:9055

it opens firefox, than vba editor start not responding, then after a few 
seconds throw error.

```

Original issue reported on code.google.com by `sava...@gmail.com` on 4 Sep 2014 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
It's most likely that you have a another program blocking the communication 
once the browser is launched.
Have you checked that your firewall or anti-spyware is not the cause of this 
issue?
```

Original comment by `florentbr` on 8 Sep 2014 at 5:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I have Kaspersky internet security 2015 with firewall turned on, I completely 
stopped kaspersky, and also stopped windows firewall and still same problem 
occurs, I also closed some programs and tried again but same error occurs.
```

Original comment by `sava...@gmail.com` on 8 Sep 2014 at 11:32

**Comment by GoogleCodeExporter** [comment]
```
Cause: Firefox WebDriver is incompatible with firefox 32?!
```

Original comment by `sava...@gmail.com` on 8 Sep 2014 at 12:16

Attachments:
- [Screenshot (101).png](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-91/comment-3/Screenshot %28101%29.png)

**Comment by GoogleCodeExporter** [comment]
```
Yes it looks like Selenium is not compatible with FF32:
http://code.google.com/p/selenium/issues/detail?id=7642

It will be fixed in version 2.43 which should be released soon.
```

Original comment by `florentbr` on 8 Sep 2014 at 8:56

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.20.0
```

Original comment by `florentbr` on 18 Sep 2014 at 4:30
- Changed state: **Fixed**
